core-rust/node-http-apis/engine-state-api-schema.yaml

```diff
@@ -0,0 +1,2886 @@
+openapi: 3.1.0
+
+info:
+  # Should be the release version, and the release display name
+  version: 'v0.0.1'
+  title: 'Engine State API'
+  license:
+    name: The Radix License, Version 1.0
+    url: https://www.radixfoundation.org/licenses/license-v1
+  x-logo:
+    url: https://docs-babylon.radixdlt.com/main/_images/radix-logo-banner.png
+    href: https://docs.radixdlt.com
+  description: |
+    This API provides a complete view of the current ledger state, operating at a relatively low
```

Comment:
We need to work out what our narrative here is - e.g. is this a beta? is it subject to breaking changes still? should people avoiding using it in prod?

Reply:
Conclusion: Small focused PR for Russ and I to summarise what beta means.